addon/templates/components/bootstrap-checkbox.hbs

```diff
@@ -21,10 +21,10 @@
     }}
   {{else}}
     {{control.label for=control.inputId label=label srOnly=srOnly}}
-    {{bootstrap/-input
+    {{bootstrap/-checkbox
       tabindex=tabindex
       checked=isChecked
-      onClick=(action "change")
+      onclick=(action "change")
```

Comment:
There is no `change` action in this component. This should be spitting an error into your console.